pkg/obfuscate/obfuscate.go

```diff
@@ -105,7 +105,13 @@ type StatsClient interface {
 type SQLConfig struct {
 	// TableNames specifies whether the obfuscator should also extract the table names that a query addresses,
 	// in addition to obfuscating.
-	TableNames bool
+	TableNames bool `json:"table_names"`
+
+	// CollectCommands specifies whether the obfuscator should extract and return commands as SQL metadata when obfuscating.
```

Comment:
Do we need to be more clear here on what a "command" is? Or is "command" a valid SQL spec term?

Reply:
I believe SQL commands is a valid SQL spec term, but I can add a quick clarifying comment for more context.

Reply:
https://github.com/DataDog/datadog-agent/pull/10042/commits/f79b990dca3b4d04d4034f7442467ce711c14553

Reply:
If it is, that's fine. No need too. Thanks 🙏🏻